src/layerInfos/MobzLayerInfo/MobzLayerInfo.js

```diff
@@ -0,0 +1,263 @@
+          nahen Einzugsgebiet sicher.
+        </span>
+      ),
+      it: (
+        <span>
+          Stazioni ferroviarie cittadine: Questa categoria è una sottocategoria
+          degli hub d’agglomerato. Le stazioni urbane svolgono anche
+          un’importante funzione di collegamento nelle grandi città. A
+          differenza degli hub d’agglomerato, le stazioni urbane garantiscono il
+          collegamento soprattutto a livello locale (quartiere urbano) per un
+          bacino d’utenza vicino.
+        </span>
+      ),
+    },
+    'Zentrum-Hubs': {
+      de: (
+        <span>
+          Zentrums-Hubs: Verkehrsdrehscheiben in mittelgrossen Städten / Zentren
+          von mittelgrossen Agglomerationen, Anbindung an hochwertigen Orts- und
+          Nahverkehr.
+        </span>
+      ),
+      fr: (
+        <span>
+          Hubs centraux: plaques tournantes du trafic dans des villes/centres de
+          taille moyenne au sein d’agglomérations également moyennes; connexion
+          avec les transports locaux et suburbains de qualité.
+        </span>
+      ),
+      en: (
+        <span>
+          Zentrums-Hubs: Verkehrsdrehscheiben in mittelgrossen Städten / Zentren
+          von mittelgrossen Agglomerationen, Anbindung an hochwertigen Orts- und
+          Nahverkehr.
+        </span>
+      ),
+      it: (
+        <span>
+          Hub centrali: nodi di traffico in città di medie dimensioni / centri
+          di agglomerati di medie dimensioni, collegamento di qualità al
+          traffico locale e urbano.
+        </span>
+      ),
+    },
+    [regionalKey]: {
+      de: (
+        <span>
+          Regional-Hubs: Verkehrsdrehscheiben von regionaler Bedeutung.
+          Anschlussmobilitätsangebote für die Bedienung des Einzugsgebietes
+          werden an diesen Standorten konzentriert und das Einzugsgebiet
+          vergrössert sich langfristig.
+        </span>
+      ),
+      fr: (
+        <span>
+          Hubs régionaux: plaques tournantes du trafic d’importance régionale;
+          interconnexion des offres de mobilité pour la desserte du bassin
+          versant concentrée sur ces sites; augmentation de l’étendue du bassin
+          versant à long terme.
+        </span>
+      ),
+      en: (
+        <span>
+          Regional-Hubs: Verkehrsdrehscheiben von regionaler Bedeutung.
+          Anschlussmobilitätsangebote für die Bedienung des Einzugsgebietes
+          werden an diesen Standorten konzentriert und das Einzugsgebiet
+          vergrössert sich langfristig.
+        </span>
+      ),
+      it: (
+        <span>
+          Hub regionali: nodi di traffico di importanza regionale. In tali sedi
+          verranno concentrate offerte di mobilità per collegare il bacino
+          d’utenza, il quale si estenderà nel tempo.
+        </span>
+      ),
+    },
+  };
+
+  return (
+    <div className="wkp-mobz-layer-info">
+      <div>
+        {img}
+        {desc[category][language]}
```

Comment:
As long as the description does not contain html tag, I would prefer to get the (translated) description from the translation files, such as in https://github.com/geops/trafimage-maps/blob/master/src/layerInfos/NetzkarteTopicInfo/NetzkarteTopicInfo.js#L15